crypto/currency-exchange/README.md

```diff
@@ -0,0 +1,137 @@
+# Currency Exchange
+
+## Firestore Collections
+
+- Currency
+  > this will contain details about the accepted currencies at crypto-website
+
+## Currency Object
+
+```
+{
+    "id": <currency_id>,
+    "name": <currency_name>,
+    "quantity": <available_quantity>
+}
+```
+
+## Currency Exchange Object
+
+```
+{
+    from: {
+        "id": <currency_id>,
+        "name": <currency_name>
+    },
+    to: {
+        "id": <currency_id>,
+        "name": <currency_name>
+    },
+    value: <exchange_value>
+}
+```
+
+## Exchange Request Object
+
+```
+{
+  from: <src_currency_id>,
+  to: <trgt_currency_id>,
+  quantity: <selected_quantity>
+}
+```
+
+## **Requests**
+
+|                         Route                         |                                Description                                 |
+| :---------------------------------------------------: | :------------------------------------------------------------------------: |
+|      [GET /currency-list](#get-currency-listing)      |     Returns all accepted currency with their availability in the bank      |
+| [GET /currency-exchange-rate](#get-currency-exchange) | Returns currency with their current exchange value with the other currency |
+|           [PATCH /convert/](#patch-convert)           |           Update the user wallet and bank currency availability            |
```

Comment:
'convert' sounds too generic. Can we get a specific name for this like above please?

Reply:
Should this be a PATCH or a POST?